lib/tree_stand/node.rb

```diff
@@ -194,9 +287,14 @@ def text
     # @overload method_missing(method_name, *args, &block)
     #   @raise [NoMethodError]
     def method_missing(method, *args, &block)
-      return super unless @fields.include?(method.to_s)
-
-      TreeStand::Node.new(@tree, T.unsafe(@ts_node).public_send(method, *args, &block))
+      if @fields.include?(method.to_s) || THINLY_WRAPPED_METHODS.include?(method)
```

Comment:
We should update `#respond_to_missing?` to reference `THINLY_WRAPPED_METHODS` too.